dali/operators/reader/tfrecord_reader_op.cc

```diff
@@ -27,16 +27,17 @@ DALI_REGISTER_OPERATOR(_TFRecordReader, TFRecordReader, CPU);
 DALI_SCHEMA(_TFRecordReaderBase)
   .DocStr(R"code(Read sample data from a TensorFlow TFRecord file.)code")
   .AddArg("path",
-      R"code(List of paths to TFRecord files.)code",
+      R"code(List of paths to the TFRecord files.)code",
       DALI_STRING_VEC)
   .AddArg("index_path",
-      R"code(List of paths to index files (1 index file for every TFRecord file).
-Index files may be obtained from TFRecord files using
-`tfrecord2idx` script distributed with DALI.)code",
+      R"code(List of paths to index files, and there is one index file for every TFRecord file.
+
+The index files can be obtained from TFRecord files by using the ``tfrecord2idx`` script
+that is distributed with DALI.)code",
       DALI_STRING_VEC);
 
 DALI_SCHEMA(_TFRecordReader)
-  .DocStr(R"code(Read sample data from a TensorFlow TFRecord file.)code")
+  .DocStr(R"code(Reads the sample data from a TensorFlow TFRecord file.)code")
```

Comment:
That would be ok if there was one sample.
```suggestion
  .DocStr(R"code(Reads sample data from a TensorFlow TFRecord file.)code")
```
or
```suggestion
  .DocStr(R"code(Reads samples from a TensorFlow TFRecord file.)code")
```

Reply:
Done